bika/lims/browser/client/ajax.py

```diff
@@ -5,25 +5,36 @@
 # Copyright 2018 by it's authors.
 # Some rights reserved. See LICENSE.rst, CONTRIBUTORS.rst.
 
-import plone, json
+import json
 
+import plone
```

Comment:
Whitespace between these two imports. I suggest to remove it.

Reply:
I thought so as well, but somehow `isort` is keeping always adding that blank line between those two...